### DISCUSSION TITLE
Launching problem

### DISCUSSION THREAD

**Discussion by monokuma11037** [X2CommunityCore/xcom2-launcher#265] [Help]
I'm using the latest version of the AML
somehow every time when I launching the game, the only things it opened is the official launcher
It has the following problem
1. if I don't set "close after launch" there will be no responded 
2. even if I set it as follow, the only thing it does is open the official launcher
3. then if I don't check the mod with the official launcher, the game won't launch properly 
4. And then, of course, some of the mod won't work with the official launcher

**Comment by RevZero** [comment]
It is kind of weird that it starts the default launcher. Post your `settings.json` and `aml.log` from the AML folder and the `C:\Users\...\Documents\My Games\XCOM2 War of the Chosen\XComGame\Logs\Launch.log`.

Also consider asking for help on the XCOM 2 discord server.

**Comment by monokuma11037** [comment]
[xcom pack.zip](https://github.com/X2CommunityCore/xcom2-launcher/files/5820975/xcom.pack.zip)

**Comment by monokuma11037** [comment]
> scord

I just post it

**Comment by nfma** [comment]
@monokuma11037 your zip file is marked as infected, best to upload the content to a pastebin and add here the links.

**Comment by RevZero** [comment]
I do not think this is AML related. Maybe someone from Discord can comment on the Launch.log that was posted in the ZIP.  If there is no further feedback I will close this soon.

**Comment by monokuma11037** [comment]
[AML.log](https://github.com/X2CommunityCore/xcom2-launcher/files/5858470/AML.log)
[Launch.log](https://github.com/X2CommunityCore/xcom2-launcher/files/5858472/Launch.log)

**Comment by monokuma11037** [comment]
[settings.json.txt](https://github.com/X2CommunityCore/xcom2-launcher/files/5858479/settings.json.txt)

**Comment by monokuma11037** [comment]
> I do not think this is AML related. Maybe someone from Discord can comment on the Launch.log that was posted in the ZIP. If there is no further feedback I will close this soon.

sorry 
I was a little bit busy working these days
I just post the file up
I think this time it will be fine

**Comment by RevZero** [comment]
Did you resolve your problem? I will move this to discussions section for now, as this is probably no issue directly related to AML.
